### PR TITLE
Manejo de compras

### PR DESCRIPTION
**Pull request by JaqKent**
## Summary

[Change!] Describe your feature, problems you had, notes, improvements and others.

## Screenshots

[Change!] Show the screenshots of the views you modified.

## Trello Card

[Change!] Link to the associated Trello card.

### CODE REVIEW

File: src/services/sells.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import api from "../config/api";
+import { Product } from "../interfaces/Products";
+
+const SELLS_PATH = "/sells";
+
+// Generated by https://quicktype.io
```
Comment (nathan-hyan) [inline src/services/sells.ts:6]: Remove comment

File: src/services/sells.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import api from "../config/api";
+import { Product } from "../interfaces/Products";
```
Comment (nathan-hyan) [inline src/services/sells.ts:2]: Remove product

File: src/services/sells.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  message: string;
+}
+
+export interface UploadResponse {
+  success: boolean;
+  links: string[];
+}
+
+export const getAllSells = () =>
+  api.get<any>(`${SELLS_PATH}/get`);
```
Comment (nathan-hyan) [inline src/services/sells.ts:19]: Replace any with Purchase[]

File: src/services/sells.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+  export const toggleFinished = (id:string) =>
+  api.put<any>(`${SELLS_PATH}/toggleFinished/${id}`);
```
Comment (nathan-hyan) [inline src/services/sells.ts:22]: Apply APIResponse in Any

**Review by nathan-hyan — CHANGES_REQUESTED** [review]

File: src/components/CustomPagination/index.tsx [hunk 5]
```diff
@@ -5,8 +5,8 @@ import { Product } from 'interfaces/Products';
 import {Pagination} from 'react-bootstrap';
 
 interface Props {
-  items?: Product[];
-  onChangePage: (items: Product[]) => void;
+  items?: any;
+  onChangePage: (items: any) => void;
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/CustomPagination/index.tsx:9]: done

File: src/components/NavigationBar/index.tsx [hunk 6]
```diff
@@ -23,6 +23,9 @@ function NavigationBar() {
               <Navbar.Text>Productos</Navbar.Text>
             </Link>
           )}
+          <Link  className='px-3 text-decoration-none' to="/buymanagement">
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/NavigationBar/index.tsx:26]: done

File: src/screens/Home/index.tsx [hunk 7]
```diff
@@ -2,6 +2,7 @@ import { faDoorOpen } from "@fortawesome/free-solid-svg-icons";
 import { useHistory } from "react-router-dom";
 import ButtonWithIcon from "components/ButtonWithIcon";
 import ContentContainer from "components/ContentContainer";
+import Management from "screens/buyManagement";
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/screens/Home/index.tsx:5]: ok

File: src/screens/buyManagement/components/constants.ts [hunk 8]
```diff
@@ -0,0 +1,236 @@
...
+          "name": "Test 334",
+          "whatsApp": 123456789
+        },
+        "amount": 22,
+        "createdAt": "2021-08-26T02:00:38.509Z",
+        "updatedAt": "2021-08-26T02:00:38.509Z",
+        "finished":true
+      },
+   
+]
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/screens/buyManagement/components/constants.ts:236]: done

**Pull request merged**